scripts/git/prune_old_tags.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+
+# Copyright 2021 The IREE Authors
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+# Deletes all but the input number of tags and corresponding releases, starting
+# with the oldest tags.
```

Comment:
Can you include a sample usage comment here to show how the args are passed?

Like what this file has: https://github.com/google/iree/blob/53f2061277d8d7b0f18be4f1c3c5c122dcff1d27/scripts/git/summarize_changes.sh#L9-L18